### ISSUE TITLE
Possible issue about RetryPolicy after upgrading to 1.2.1

### ISSUE THREAD

**Issue by barbarosalp** [StackExchange/StackExchange.Redis#579]
Hi,

We have been using CacheManager.net with Redis Backplane, yesterday we updated stackexchange to 1.2.1 from a previous version.

This morning we started getting Redis Connection errors until we got back to a previous release.
Can it be about the latest version of StackExchange.redis?

There are plenty of error messages like below, all of them are about Retry...

```
Call Stack
Show Just My Code
METHOD
StackExchange.Redis.RedisConnectionException
StackExchange.Redis.ConnectionMultiplexer.ExecuteSyncImpl
StackExchange.Redis.RedisBase.ExecuteSync
CacheManager.Redis.RedisCacheHandle`1.Eval
CacheManager.Redis.RedisCacheHandle`1+<>c__DisplayClass43_0.<GetCacheItemAndVersion>b__0
CacheManager.Redis.RetryHelper.Retry
CacheManager.Redis.RedisCacheHandle`1.Retry
CacheManager.Redis.RedisCacheHandle`1.GetCacheItemAndVersion
CacheManager.Redis.RedisCacheHandle`1.GetCacheItemInternal
CacheManager.Core.BaseCacheManager`1.GetCacheItemInternal
CacheManager.Core.Internal.BaseCache`1.Get
[external code]
System.Web.HttpApplication.ExecuteStep
```

Does anyone get into this issue like us after updating to the latest version of StackExchange.Redis?

Thanks

**Comment by NickCraver** [comment]
There was a major build regression in 1.2.1 which should be fixed in 1.2.5 onwards. Can you please confirm whether or not you are seeing issues on releases since then?

**Comment by NickCraver** [comment]
No activity here, so closing out to cleanup.
